CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - When the kernel supports unprivileged overlay mounts in a user
   namespace, the container will be constructed using an overlay
   instead of underlay layout.
+- Define EUID in %environment alongside UID
```

Comment:
Ah, I'm sorry I missed the category this was in.  Even though it really isn't a bug fix, please put this instead down in the bug fix section.  That's because it is going to get backported to the release-1.1 branch where we'll call it "Changes in this release".  Then after the release it will get removed from the Bug fixes in the main branch.

While you're in there, please add a period like the other bullet points.